plugins/hls-explicit-imports-plugin/test/Main.hs

```diff
@@ -79,18 +123,16 @@ caTitle _                         = Nothing
 -- code lens tests
 
 codeLensGoldenTest :: FilePath -> Int -> TestTree
```

Comment:
maybe these things could also go in `hls-test-utils`? Seems like you've been repeating something similar a few times. Good to deduplicate the test stuff too!